### ISSUE TITLE
javax.jms Dependency removed from the product 

### ISSUE THREAD

**Issue by Buddhimah** [wso2/product-is#18542]
**Describe the issue:**

In the ealier IS versions we had the  javax.jms; version="[1.1.0,2.0.0)" in the product runtime. This was used for the webapp deployments as well. However since it was removed recently from IS 7.00 RC1 following issues occured 

/api#onprem-userstore webapp deployment fails due to   Unresolved requirement: Import-Package: javax.jms; version="[1.1.0,2.0.0)"

This dependency was packed with geronimo-spec-jms_1.1.0.rc4-wso2v1.jar seems this was removed when removing  the workflow feature from identity server 

**How to reproduce:**

**Expected behavior:**

**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
 - Product Version: [e.g., IS 5.10.0, IS 5.9.0]
 - OS: [e.g., Windows, Linux, Mac]
 - Database: [e.g., MySQL, H2]
 - Userstore: [e.g., LDAP, JDBC]

---

### Optional Fields

**Related issues:**

**Suggested labels:**

**Comment by madurangasiriwardena** [comment]
@Buddhimah,

The `api#onprem-userstore` you have mentioned should be a custom web app deployed within the IS. We have removed this dependency in the process of making the workflow as a connector. So you will have to make sure you pack this dependency with your web app.

This information needs to be available in the migration guidelines. Following jars were removed with the workflow connector.

- org.objectweb.asm.commons
- org.objectweb.asm.tree
- geronimo-spec-jms
